src/01-general/users.apib

```diff
@@ -40,8 +40,11 @@ Get the current authenticated user.
             + time_zone: `Europe/Brussels` (string)
             + preferences (object)
                 + invoiceable: true (boolean)
-                + time_tracking_freezing: true (boolean)
-                + time_tracking_freezing_cutoff_days: 7 (number)
+                + historic_time_tracking_limit (object, nullable)
+                    + unit (enum)
+                        + Members
+                            + day - Days
+                    + value: 7 (number)
```

Comment:
PP: use `hours` as the basic unit here. An hour seems like exactly the right granularity and doesn't require us to add a new unit once we want to limit to i.e. 8 hours.

Reply:
The disadvantage of always using hours is that it doesn't directly map to the value added in the UI, which may induce some degree of uncertainty.

On the other hand, if we ever decide to introduce a different granularity (i.e. hours), having the unit here means we can also return hours and change the unit to hours at that point.

But I will change the default unit to hours if more people feel this is the right way to go.

Reply:
We can definitely add more units in the future. However residing to hours from the start will give us a lot of flexibility for the future, i.e. not having to update the mobile app if we allow the cut-off to be less than a day.

I'd even dare to say we could then ever only return the value in hours and not require any integrator to recalculate to a different unit.

Reply:
Ok, changed to hours